getenv is defined in stdlib.h. Header should be included regardless of platform

--- a/src/common/file-utils.cpp	2025-11-21 14:12:02.000000000 +0100
+++ b/src/common/file-utils.cpp	2026-08-15 14:29:46.731699754 +0200
@@ -30,7 +30,6 @@
 #include "file-utils.h"
 #include "config.h"
 #ifdef _MSC_VER
-#include <stdlib.h>
 #include <stdio.h>
 #include "uniwin.h"
 #include <errno.h>
@@ -39,6 +38,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #endif // !_MSC_VER
+#include <stdlib.h>
 #include "str-utils.h"
 #include <algorithm>
 #ifdef _WIN32
